store/src/server/api.rs

```diff
@@ -174,6 +176,43 @@ impl api_server::Api for StoreApi {
             nullifiers: convert(nullifiers_blocks),
         }))
     }
+
+    // TESTING ENDPOINTS
+    // --------------------------------------------------------------------------------------------
+
+    // Returns a list of all nullifiers
+    async fn list_nullifiers(
+        &self,
+        _request: tonic::Request<EmptyRequest>,
+    ) -> Result<Response<ListNullifiersResponse>, Status> {
+        let request = self.state.list_nullifiers().await.map_err(internal_error)?;
```

Comment:
I think the naming of the variable (i.e., `request`) is a bit confusing here - since what we get back is a list of nullifiers.